drwmutex.go

```diff
@@ -41,6 +41,7 @@ func log(msg ...interface{}) {
 
 // DRWMutexAcquireTimeout - tolerance limit to wait for lock acquisition before.
 const DRWMutexAcquireTimeout = 1 * time.Second // 1 second.
+const DRWMutexInfinite = time.Duration(1<<63-1)
```

Comment:
may be `DRWMutexInfiniteTimeout`?

You would need to add a doc string so that `godoc` will show.